xclim/indices/_conversion.py

```diff
@@ -1180,3 +1181,574 @@ def potential_evapotranspiration(
 
     out.attrs["units"] = "mm"
     return amount2rate(out, out_units="kg m-2 s-1")
+
+
+@declare_units(tas="[temperature]", hurs="[]", sfcWind="[speed]", tr="[temperature]")
+def universal_thermal_climate_index(
+    tas: xr.DataArray,
+    hurs: xr.DataArray,
+    sfcWind: xr.DataArray,
+    tr: xr.DataArray = None,
+) -> xr.DataArray:
+    """
+    Mean Universal Thermal Climate Index (UTCI)
+
+    The mean of daily UTCI
+
+    Parameters
+    ----------
+    tas : xarray.DataArray
+        Mean daily temperature
+    hurs : xarray.DataArray
+        Relative Humidity
+    sfcWind : xarray.DataArray
+        Wind velocity
+    tr: xarray.DataArray, optional
```

Comment:
It seems that `tr` exists in CMIP6, I didn't know that!:

https://github.com/PCMDI/cmip6-cmor-tables/blob/cf039eb222cb0b7b92b3df38d986b8015450a828/Tables/CMIP6_Eday.json#L1997-L2006

But it has a slightly different description. Do you know how it relates to "the mean radiant temperature" this indice expects? Are they close enough so that we could use `tr` if it was available from a model?
If not, I would prefer using a non-cmip6 name to avoid any confusion.  `tmrt` and `tmr` are available.

Reply:
[Here](http://www.utci.org/utci_doku.php) you can find the description and calculation of `tr`. The CMIP6 `tr` is not the correct radiative temperature. The 'Effective radiative surface temperature' (tr), in the CMIP6 var list, is the temperature associated with a black body (Earth) and used for the radiative transfer calculations. In the UTCI equation, the 'radiative temperature' we need is the [solar radiation](https://doi.org/10.1016/j.wace.2018.01.004) that is directly hitting the person or bouncing back from buildings onto a person in the urban canon. Thus, I'll use the non-cmip6 name `tmrt`. Or should we delete `tr` fully from the paramter list?

Reply:
As it seems an essential element of the original equation, I'd leave `tmrt` in the indice. I'm guessing that the scope of xclim often will mean that it is not available, but at least we have the option and it makes it more obvious that not passing it _is an approximation_.